Document/0x06d-Testing-Data-Storage.md

```diff
@@ -835,9 +847,23 @@ Manufacturers want to provide device users with an aesthetically pleasing effect
 
 While analyzing the source code, look for the fields or screens that take or display sensitive data. Use [UIImageView](https://developer.apple.com/documentation/uikit/uiimageview "UIImageView") to determine whether the application sanitizes the screen before being backgrounded.
 
-The following is a sample remediation method that will set a default screenshot:
+The following is a sample remediation method that will set a default screenshot.
 
-```objectivec
+Swift:
+
+```swift
+private var backgroundImage: UIImageView?
+
+func applicationDidEnterBackground(_ application: UIApplication) {
+    let myBanner = UIImageView(image: #imageLiteral(resourceName: "overlayImage"))
+    backgroundImage = myBanner
+    window?.addSubview(myBanner)
+}
```

Comment:
Did you test this code?

Reply:
I have run it, but It doesn't work as it should, it is missing the constraints and removing this view after getting to the foreground.
I believe it was added here as a general idea how implement such feature.
I can add complete implementation :)